policies/policy-change-process.md

```diff
@@ -66,3 +69,17 @@ and closing the pull request without merging.
 
 If the policy change is approved, the pull request is merged to the
 master branch of the technical-policies repository.
+
+Exceptions
```

Comment:
As commented in https://github.com/openssl/general-policies/pull/17/files#r847852761, this is perhaps not the best title.
I suggest "Minor edits"